README.md

```diff
@@ -348,11 +348,9 @@ The `build-test-cxx` workflow supports building and running C++ tests on GitHub
 Windows, Linux, and macOS. Image details are available [here](https://github.com/actions/runner-images).
 
 The `build-test-package-python.yml` workflow supports Python package generation for the following platforms and architectures:
-- Windows 10 x86_64 platforms
 - Windows 11 x86_64 platforms
 - macOS 10.9+ x86_64 platforms
 - macOS 11.0+ ARM64 platforms
-- Linux glibc 2.17+ (E.g. Ubuntu 18.04+) x86_64 platforms
```

Comment:
Why remove AMD64 platforms?

Reply:
I was removing 18.04,   all the CI environments have been migrating to 22.04, or 24.04.  I had a script that flags the 18.04 regex to let me know to review and update.  I did not read close enough to realize that this was referencing the compatibility of the packages that are built from a docker image downloaded to a 24.04 CI environment.

   I re-added and updated the documentation and added more information to track